src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -623,6 +728,13 @@ public CourseSummaryBundle getCourseSummaryWithoutStats(String courseId) throws
         return courseList;
     }
     
+    /**
+     * Get a Map<CourseId, {@link CourseSummaryBundle} for all courses mapped to a given instructor.
```

Comment:
Describe what the map contains, perhaps something like `Gets a map of course ids to course summaries, excluding its statistics, for all courses of a given instructor.`